client/src/components/requester_order/partial-details.html

```diff
@@ -1,15 +1,14 @@
 <div id="requester-order-details">
-  <h1> Sit tight, you will have a queue hero soon! </h1>
+  <div class="container">
+    <h1 class="requester-task-stage"> Sit tight, you will have a queue hero soon! </h1>
 
-  <div>
-    <button ng-click="ctrl.cancelOrder()">Cancel Order</button>
-  </div>
-
-  <h3> Your order </h3>
-  Location: <span> {{ ctrl.order.vendor }}</span> <br/>
-  Item: <span> {{ ctrl.order.item }}</span> <br/>
-  Additional Requests: <span> {{ ctrl.order.additionalRequests }} </span> <br/>
-  Meeting time: <span> {{ ctrl.order.meetingTime | date : 'h:mm a' }} </span> <br/>
-  Price: <span> {{ ctrl.order.moneyExchanged | currency }} </span> <br/>
+    <!-- <h3> Your order </h3> -->
+    <h4 class="requester-choice"> <small> Location: </small> {{ ctrl.order.vendor }} </h4>

```

Comment:
The `<small>` tag is usually used to make comments/annotations like disclaimers, copyrights, legal stuff.

The header tags like `<h1>``<h4>` are usually used to show headings/titles to identify sections. They shouldn't be repeated like a list or to set the text size. To do that, the convention is to wrap text in a `<div>` or `<span>` and change it via CSS.